packages/transaction-controller/src/TransactionController.ts

```diff
@@ -603,12 +615,16 @@ export class TransactionController extends BaseController<
    * @param transactionId - The ID of the transaction to cancel.
    * @param gasValues - The gas values to use for the cancellation transaction.
    * @param options - The options for the cancellation transaction.
+   * @param options.actionId - The actionId passed from UI
```

Comment:
```suggestion
   * @param options.actionId - Unique ID to prevent duplicate requests.
```

Reply:
Fixed in https://github.com/MetaMask/core/pull/1894/commits/951182dd5b77ecaad6ec699a22b86a1fc9193931